src/Dto/StimulusActionsDto.php

```diff
@@ -17,57 +17,21 @@ final class StimulusActionsDto extends AbstractStimulusDto
     private $parameters = [];
 
     /**
-     * @param string      $controllerName the Stimulus controller name
-     * @param string      $actionName     the action to trigger
-     * @param string|null $eventName      The event to listen to trigger. Optional.
-     * @param array       $parameters     Parameters to pass to the action. Optional.
-     *
-     * @throws \Twig\Error\RuntimeError
+     * @param array  $parameters Parameters to pass to the action. Optional.
      */
-    public function addAction($controllerName, string $actionName = null, string $eventName = null, array $parameters = []): void
+    public function addAction(string $controllerName, string $actionName, string $eventName = null, array $parameters = []): void
```

Comment:
The addition of the `string` argument type (which i also added to the other Dto classes) is technically a BC break if you're using version 1.15.0 exactly. However, to have code that would trigger that break, you would have needed to start using these classes directly in 1.15.0 and pass the array argument... which would have triggered deprecations immediately (you don't see it here because it's already moved, but in 1.15.0 a deprecation is triggered in this method if you pass an array to `$controllerName`). So, I think it's not a problem.